### PR TITLE
Add distri export pdf

### PR DESCRIPTION
**Pull request by jaumef**
### Description:

ADD: Exportació de correus de Factures en distribuidora

### Tasks:

- [x] ADD Docs exportació PDF de factures (distri)
- [x] Update Docs facturacio 4 passes (distri)
- [x] Traduir

### Links:

* [Facturació#Facturació en 4 passos](http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/ca/distri/facturacion/#facturacio-en-4-passos)
* [Facturació#Exportació a PDF](http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/ca/distri/facturacion/#exportacio-de-pdfs)
* [Facturació#Facturación en 4 pasos](http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/ca/distri/facturacion/#facturacion-en-4-pasos)
* [Facturación#Exportación a PDF](http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/es/distri/facturacion/#exportacion-de-pdfs)

### CODE REVIEW

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/facturacion.md [hunk 1]
```diff
@@ -24,55 +24,55 @@ Si una pòlissa es dona de baixa després de haver-se facturat s'ha de comprovar
...
-El botó obrir es el primer pas a realitzar per iniciar la facturació
-d'un determinat període. Si s'intenta obrir un lot sense haver tancat
-l'anterior apareixerà un error indicant que existeix més d'un lot obert.
+    El botó obrir es el primer pas a realitzar per iniciar la facturació
+    d'un determinat període. Si s'intenta obrir un lot sense haver tancat
+    l'anterior apareixerà un error indicant que existeix més d'un lot obert.
 
-Una vegada obert un lot, les pòlisses a facturar passen del estat “borrador”
-al estat “obert”.
+    Una vegada obert un lot, les pòlisses a facturar passen del estat “borrador”
```
Comment (MiquelIR) [inline docs/distri/facturacion.md]: de l'estat "borrador" a l'estat

File: docs/distri/facturacion.md [hunk 2]
```diff
@@ -24,55 +24,55 @@ Si una pòlissa es dona de baixa després de haver-se facturat s'ha de comprovar
...
 
-#### 2 Validar:
+2. **Validar**
 
-El botó validar comprova que totes les pòlisses del lot compleixin
-els requisitos per poderse facturar. Si la validació de la pòlissa es correcta
-pasa al estat “A punt de facturar”.
+    El botó validar comprova que totes les pòlisses del lot compleixin
+    els requisitos per poderse facturar. Si la validació de la pòlissa es correcta
+    pasa al estat “A punt de facturar”.
```
Comment (MiquelIR) [inline docs/distri/facturacion.md]: els **requisits** per **poder-se** 
**passa a l'estat**

File: docs/distri/facturacion.md [hunk 3]
```diff
@@ -24,55 +24,55 @@ Si una pòlissa es dona de baixa després de haver-se facturat s'ha de comprovar
...
 
-Comprovacions que realiza:
+    Comprovacions que realiza:
```
Comment (MiquelIR) [inline docs/distri/facturacion.md]: **realitza**

File: docs/distri/facturacion.md [hunk 4]
```diff
@@ -454,6 +456,37 @@ factures generades.
...
+- Per comercialitzadora
+- Per lot de facturació
+
+I el podem trobar en els mateixos registres:
+
+- Lot de facturació
+- Factures
+
+Aquest assistent ens generarà un fitxer `.zip` que conté les diferents factures
+exportades en format PDF, utilitzant l'informe de factura sel·leccionat:
```
Comment (MiquelIR) [inline docs/distri/facturacion.md]: **seleccionat**

**Comment by giscegit** [comment]
Documentation build URL:
http://builds.gisce.net/powerp-docs/powerp_add_distri_export_pdf/

Things to review:
- [ ] Typos in Catalan
- [ ] Typos in Spanish
- [ ] Links routes in Spanish
- [ ] Links work in Catalan
- [ ] Links work in Spanish
- [ ] Images in Spanish
- [ ] Page index shows all headers

File: docs/distri/facturacion.md (hunk 1 above)
Comment (MiquelIR) [reply docs/distri/facturacion.md]: @jaumef ?

**Pull request merged**